lib/matplotlib/tests/test_colors.py

```diff
@@ -526,6 +526,25 @@ def test_Normalize():
     assert 0 < norm(1 + 50 * eps) < 1
 
 
+def test_FuncNorm():
+    def _forward(x):
+        return (x**2)
+    def _inverse(x):
+        return np.sqrt(x)
+
+    norm = mcolors.FuncNorm((_forward, _inverse), vmin=0, vmax=10)
+    expected = np.array([0, 0.25, 1])
+    assert_array_almost_equal(norm([0, 5, 10]), expected)
```

Comment:
I guess you can do something like:
```suggestion
    input = np.array([0, 5, 10])
    output = np.array([0, 0.25, 1])
    assert_array_almost_equal(norm(input), output)
    assert_array_almost_equal(norm.inverse(output), input)
```
to exercise the inverse function as well?

Reply:
Sure... I'm not particularly sure if the inverse gets used, but...